ui/src/components/WebRTCVideo.tsx

```diff
@@ -10,7 +10,7 @@ import {
   useVideoStore,
 } from "@/hooks/stores";
 import { keys, modifiers } from "@/keyboardMappings";
-import { useResizeObserver } from "@/hooks/useResizeObserver";
+import { useResizeObserver } from "usehooks-ts";
```

Comment:
use the **already included** `usehook-ts` implementations and kill off our versions